azure-pipelines.yml

```diff
@@ -863,3 +864,18 @@ jobs:
 #     fetchDepth: 25
 #   - bash: ./checker/bin-devel/test-downstream.sh
 #     displayName: test-downstream.sh
+
+- job: windows_junit_jdk21
+  dependsOn:
+    - junit_jdk21
+  pool:
+    vmImage: 'windows-latest'
+  timeoutInMinutes: 90
+  steps:
+    - checkout: self
+      fetchDepth: 25
+    - bash: ./checker/bin-devel/test-cftests-junit.sh
+      displayName: Run all JUnit tests
+      env:
+        JAVA_HOME: $(JAVA_HOME_21_X64)
```

Comment:
This fails with

````
* What went wrong:
Could not determine the dependencies of task ':compileJava'.
> Failed to calculate the value of task ':compileJava' property 'javaCompiler'.
   > Cannot find a Java installation on your machine matching this tasks requirements: {languageVersion=17, vendor=any, implementation=vendor-specific} for WINDOWS on x86_64.
      > No locally installed toolchains match and toolchain download repositories have not been configured.
````

I'm not sure why it asks for JDK 17... Let me try adding that the the PATH as well.